database/sqlite.go

```diff
@@ -0,0 +1,73 @@
+package database
```

Comment:
What do you think of renaming this to `db`? I find GoLang prefers more simple package names (will make it easier when consuming)

Reply:
imo database or storage is pretty descriptive and should keep it that way unless you say so

Reply:
Please change it to `db`. It's a very common abbreviation for database and I prefer making it concise, thanks!

Reply:
changed